### ISSUE TITLE
The 0.2.1 wheels for Python 3.11 only seem to be available for Linux

### ISSUE THREAD

**Issue by jrobbins-LiveData** [MattFanto/orjson-ddb#7]
On Windows 10, my `poetry update` command is failing with

```
poetry update
Updating dependencies
Resolving dependencies...

Package operations: 1 install, 0 updates, 0 removals

  • Installing orjson-ddb (0.2.1)

  RuntimeError

  Unable to find installation candidates for orjson-ddb (0.2.1)
```

The PyPi "Download Files" page seems to show that the only Python 3.11 `.whl` file is for Linux.

<img width="451" alt="image" src="https://github.com/MattFanto/orjson-ddb/assets/79278744/904feb2c-2b8a-4c30-9590-7eb874bd72c8">

**Comment by ppena-LiveData** [comment]
Is it as simple as adding `'3.11'` to this list in [.github/workflows/windows.yaml](https://github.com/MattFanto/orjson-ddb/blob/da0006181d5ada88d3dadc09c9b19a4be1ebcd31/.github/workflows/windows.yaml#L14)?
```yaml
python-version: ['3.7', '3.8', '3.9', '3.10']
```

**Comment by MattFanto** [comment]
Hi both, I missed this issue when initially open. I released 0.2.2 where you can find the release for Python3.11 on windows.
Let me know if you face any issue and I can check

**Comment by ppena-LiveData** [comment]
Looks great, thanks!
